dot/telemetry/mailer.go

```diff
@@ -0,0 +1,132 @@
+package telemetry
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/ChainSafe/gossamer/internal/log"
+	"github.com/ChainSafe/gossamer/lib/genesis"
+	"github.com/gorilla/websocket"
+)
+
+const (
+	defaultMessageTimeout = time.Second
+	defaultMaxRetries     = 5
+	defaultRetryDelay     = time.Second * 15
```

Comment:
Please inline this one as a constant in `BootstrapMailer` since it's only used there. You could place it right before passing it to the timer. Also rename it, it's not really a `default`.

Same comment for the other 2 constants above